dart/dynamics/InverseKinematics.h

```diff
@@ -478,7 +559,7 @@ class InverseKinematics : public common::Subject
 
     /// Constructor
     explicit JacobianTranspose(InverseKinematics* _ik,
-                               double _clamp = DefaultIKGradientComponentClamp);
+                               const Properties& properties = Properties());
```

Comment:
You're right. One option would be to bring back the constructor that I've removed and mark it as deprecated. If we plan on having a 5.2 release, then I definitely think the ``Analytical`` content should go in it.